src/LanguageServer.ts

```diff
@@ -194,7 +194,7 @@ export class LanguageServer {
                 completionProvider: {
                     resolveProvider: true,
                     //anytime the user types a period, auto-show the completion results
-                    triggerCharacters: ['.'],
+                    triggerCharacters: ['.', ':'],
```

Comment:
@chrisdp  @elsassph @georgejecook  I added a colon to the trigger characters. This means anytime you type a colon _anywhere_ in your code, it will trigger completions. The typescript language server doesn't do this, but it was the only way to get `"pkg:` inside a string to trigger these completions automatically. 

The side-effect of this is that it triggers the global completions inside of associative arrays too (after typing the colon). I'm not sure that's actually a bad thing, but I just wanted to get your feedback on whether that would be annoying or acceptable.

Reply:
Hummmm my gut tells me that could be very annoying to encounter. I would need to likely test it in practice to get a feel but there are quite a few times where I have completions I have to dismiss. Knowing when to stop suggesting is something that could still be improved so I think this will just add to that. 

I would likely rather be ok with just needing to hit the completion hotkey for now.

Reply:
I normally would like that behaviour, but for AAs it may be annoying to have it pop, especially because the completion list will disappear as soon as you press a space for instance, which means it's going to pop for no useful reason.

Reply:
Yeah, i agree. it would be nice to have the completions show for this, but it's not worth the side-effects. I removed the colon trigger character.